### PR TITLE
Dockerfile integration and other minor changes

### PR DESCRIPTION
**Pull request by Faithtosin**
**Motivation**
Added a Dockerfile to handle deployment automation.  

Needed to upgrade node-sass and pass just an arg to computeSyncPeriodAtSlot to get the build going. 

**Description**

Closes #issue_number

**Steps to test or reproduce**

### CODE REVIEW

**Review by wemeetagain — APPROVED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    libxss1 \
+    libxtst6 \
+    ca-certificates \
+    fonts-liberation \
+    libappindicator1 \
+    libnss3 \
+    lsb-release \
+    xdg-utils \
+    wget \
+    libgbm1
```
Comment (dapplion) [inline Dockerfile:51]: Why are all this lib* dependencies necessary?

Comment (Faithtosin) [reply Dockerfile:51]: Puppeteer needs them to work. [This is the full list of dependencies](https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#:~:text=Debian%20(e.g.%20Ubuntu)%20Dependencies)

Comment (dapplion) [reply Dockerfile:51]: Why do you need Puppeteer for this deployment?

Comment (Faithtosin) [reply Dockerfile:51]: After `yarn build`, the postbuild process runs Puppeteer.  This is the current set behaviour after `yarn build` is called.

Comment (dapplion) [reply Dockerfile:51]: Oh to pre-render right :+1:

**Review by dapplion — APPROVED** [review]

**Review by wemeetagain — APPROVED** [review]

**Comment by dapplion** [comment]
@Faithtosin Could we have some CI to ensure this all works?

**Pull request merged**